//Creaqcion del Try - Catch

console.log(1 + 1);

try{
    autenticarUsuarios();
}catch(error){
    console.log(error);
}

console.log(2 + 2 );